.travis/python.release.sh

```diff
@@ -16,63 +16,17 @@
 set -e -x
 
 # Release:
-# docker run -i -t --rm -v $PWD:/v -w /v --net=host ubuntu:14.04 /v/.travis/python.release.sh
-
-export BAZEL_VERSION=0.20.0 BAZEL_OS=linux
-
-DEBIAN_FRONTEND=noninteractive apt-get -y -qq update
-DEBIAN_FRONTEND=noninteractive apt-get -y -qq install \
-  software-properties-common > /dev/null
-
-DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:deadsnakes/ppa
-
-DEBIAN_FRONTEND=noninteractive apt-get -y -qq update
-DEBIAN_FRONTEND=noninteractive apt-get -y -qq install \
-  gcc g++ make patch \
-  python \
-  python3 \
-  python3.5 \
-  python3.6 \
-  unzip \
-  curl > /dev/null
-
-curl -sOL https://github.com/bazelbuild/bazel/releases/download/${BAZEL_VERSION}/bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh
-chmod +x bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh
-
-# Install bazel, display log only if error
-./bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh 2>&1 > bazel-install.log || (cat bazel-install.log && false)
-rm -rf bazel-${BAZEL_VERSION}-installer-${BAZEL_OS}-x86_64.sh
-rm -rf bazel-install.log
-
-curl -OL https://nixos.org/releases/patchelf/patchelf-0.9/patchelf-0.9.tar.bz2
-tar xfa patchelf-0.9.tar.bz2
-(cd patchelf-0.9 && ./configure --prefix=/usr && make && make install)
-rm -rf patchelf-0.9*
-curl -sOL https://bootstrap.pypa.io/get-pip.py
-python3.6 get-pip.py
-python3.5 get-pip.py
-python3 get-pip.py
-python get-pip.py
-rm -rf get-pip.py
-python3 -m pip install -q auditwheel==1.5.0
-python3 -m pip install -q wheel==0.31.1
-
-if [[ ! -z ${TENSORFLOW_INSTALL} ]]; then
-  python -m pip install -q ${TENSORFLOW_INSTALL}
+# docker run -i -t --rm -v $PWD:/v -w /v --net=host buildpack-deps:14.04 /v/.travis/python.release.sh
+
+export TENSORFLOW_INSTALL="$(python setup.py --package-version)"
+export PYTHON_VERSION="python python3 python3.5 python3.6"
```

Comment:
@BryanCutler The build of whl that is pushed to PyPi.org, has to run on Ubuntu 14.04 as otherwise the `pip install tensorflow-io` will not work with `pip install tensorflow'.

If we build whl in Ubuntu 18.04, then some users that is using an old version of Ubuntu (16.04), will find out that they could run tensorflow but not tensorflow-io. This will not be a good experience for them.

So we stick with Ubuntu 14.04 for building whl. And the generated binary will be tested on Ubuntu 16.04/18.04.

On the other hand, some developers may not be interested in building whl themselves. They only want to build the repo and run locally, maybe debug something. They likely will not use Ubuntu 14.04 (too old). That is why we also test building the source with Ubuntu 16.04/18.04.